lte/gateway/c/core/oai/tasks/sgw/spgw_state_manager.cpp

```diff
@@ -112,8 +114,7 @@ status_code_e SpgwStateManager::read_ue_state_from_db() {
       return RETURNerror;
     }
     OAILOG_DEBUG(log_task, "Reading UE state from db for key %s", key.c_str());
-    spgw_ue_context_t* ue_context_p =
-        (spgw_ue_context_t*)calloc(1, sizeof(spgw_ue_context_t));
+    spgw_ue_context_t* ue_context_p = new spgw_ue_context_t();
```

Comment:
Null check for ue_context_p

Reply:
Only if we want to log the error but in any case the application should be stopped.

Reply:
Then it's good to catch the exception to understand where the application is going wrong. @ssanadhya May I know your thoughts on this?

Reply:
@rsarwad , since this would be a system error, it is best to let the process terminate as a result of `bad_alloc` exception. No need to catch that exception.